addons/website_sale/static/src/js/website_sale_validate.js

```diff
@@ -39,7 +39,7 @@ publicWidget.registry.websiteSaleValidate = publicWidget.Widget.extend({
                 } else {
                     var $message = $(result.message);
                     var $warning =  $("<i class='fa fa-warning' style='margin-right:10px;'>");
-                    $warning.attr("title", _t("We are waiting the confirmation of the bank or payment provider"));
+                    $warning.attr("title", _t("We are waiting for confirmation of the bank or the payment provider"));
```

Comment:
```suggestion
                    $warning.attr("title", _t("We are waiting for confirmation from the bank or the payment provider"));
```

Then I suggest to modify this as well )

Reply:
I agree. I'll do a change later today =)